src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1792,7 +1792,7 @@ java_bytecode_parsert::parse_method_handle(const method_handle_infot &entry)
     //                "new" when it is a class variable, instantiated in <init>
     lambda_method_handle.lambda_method_name =
       name_and_type.get_name(pool_entry_lambda);
-    lambda_method_handle.lambda_method_ref = method_ref;
+    lambda_method_handle.lambda_method_ref = std::move(method_ref);
```

Comment:
@mgudemann You requested this change - what is the reason? I think this is introducing a risk that someone tries to use `method_ref` introducing undefined behaviour. A unnecessary copy of a small string doesn't seem worth the risk?

Reply:
agreed